src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="upper__form" 
+      action="#" 
+      method="get">
```

Comment:
```suggestion
    <form 
      class="upper__form" 
      action="#" 
      method="get"
    >
```

Reply:
Apologies, I just noticed the figma prototype being available - will fix shortly.